lib/Mojo/Cookie/Response.pm

```diff
@@ -123,6 +126,17 @@ cookie.
 
 Max age for cookie.
 
+=head2 samesite
+
+  my $samesite = $cookie->samesite;
+  $cookie      = $cookie->samesite('Lax');
+  # or
```

Comment:
This is inconsistent with the rest of the docs. We just show calling conventions at the top, then a description, and then more detailed examples with descriptive comments.